tests/should-fail/iati-activities/iati-activity/result/document-link/title/err-not-iati-xml-forbidden-attribute_non-permitted-attribute-present.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity>
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx">
+      <narrative>Organisation name</narrative>
+    </reporting-org>
+    <title>
+      <narrative>Xxxxxxx</narrative>
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative>
+    </description>
+    <participating-org role="xx"></participating-org>
+    <activity-status code="xx"/>
+    <activity-date type="xx" iso-date="2013-11-27"/>
+    <activity-date type="xx" iso-date="2013-11-27">
+      <narrative>Some stuff ere</narrative>
+    </activity-date>
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <document-link url="xx" format="xx">
+        <title additional-attribute="xx"><!-- FAIL: An addiitonal attribute added. -->
+          <narrative>Sample text</narrative>
+        </title>
+        <category code="xx" />
```

Comment:
remove unnecessary whitespace

Reply:
Is there unnecessary whitespace - I can't see any...

![image](https://user-images.githubusercontent.com/8247168/33430945-42ead668-d5ca-11e7-921a-1983f200f4b7.png)